src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,114 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { PaginationRules } from '../../types/PaginationRules';
+
+interface Props<T> {
+  paginationRules: PaginationRules
+  items: T[]
+  onPageChanged: (rules: PaginationRules) => void
+}
+
+function createLinksList(amount: number,
+  activePage: number,
+  onClickCallBack: (pageNumber: number) => void) {
+  const linksList = [];
+
+  // eslint-disable-next-line no-plusplus
```

Comment:
Need lock its eslint rule?

Reply:
yes, otherwise it says `Unary operator '++' used.`